packages/client/hmi-client/src/components/workflow/model-operation.ts

```diff
@@ -15,5 +15,8 @@ export const ModelOperation: Operation = {
 		model: Model;
 		intialValues: StringValueMap;
 		parameterValues: StringValueMap;
-	}) => [{ type: 'modelConfig', value: modelConfig }]
+	}) => {
+		modelOperation.outputs.push({ type: 'modelConfig' });
+		return [{ type: 'modelConfig', value: modelConfig }];
+	}
```

Comment:
There is a misunderstanding here. The ModelOperations' output does not change, so the cloneDeep you have elsewhere is not necessary.

It is kind of like saying: Tim Hortons outputs coffee, but how many pick up windows of coffee is a property of each location, and not that of the definition of the franchise.